Atomic/storage.py

```diff
@@ -184,7 +184,11 @@ def modify(self):
                         cmd = 'bash -c ". /usr/lib/docker-storage-setup/libdss.sh; check_data_size_syntax {0}"'.format(self.args.lvsize)
                         util.check_call(cmd)
                     except subprocess.CalledProcessError:
-                        raise ValueError("Invalid format for --lvsize")
+                        try:
+                            cmd = 'bash -c ". /usr/share/container-storage-setup/libcss.sh; check_data_size_syntax {0}"'.format(self.args.lvsize)
```

Comment:
Wow, man this code was evil :japanese_ogre:.  Maybe a bit better to do something like:

```
if os.path.isdir('/usr/share/container-storage-setup'): 
  libcssdir='/usr/share/container-storage-setup'
else:
  libcssdir='/usr/lib/docker-storage-setup'
cmd = 'bash -c ".' + libcssdir + '/libcss.sh' ...
```
?